docs/developers/debugging/releases.md

```diff
@@ -93,16 +93,16 @@ required before these github actions will work on your fork and development
 branch.
 
 To run
-[`schedule_snapshot_release.yml`](https://github.com/iree-org/iree/blob/main/.github/workflows/schedule_snapshot_release.yml),
+[`schedule_snapshot_release.yml`](https://github.com/openxla/iree/blob/main/.github/workflows/schedule_snapshot_release.yml),
 comment out
-[this line](https://github.com/iree-org/iree/blob/392449e986493bf710e3da637ebf807715da9ffe/.github/workflows/schedule_snapshot_release.yml#L14):
+[this line](https://github.com/openxla/iree/blob/392449e986493bf710e3da637ebf807715da9ffe/.github/workflows/schedule_snapshot_release.yml#L14):
 ```yaml
 # Don't run this in everyone's forks.
-if: github.repository == 'iree-org/iree'
+if: github.repository == 'openxla/iree'
```

Comment:
Huh this documentation is out of date. I removed that line. I'll clean that up in a follow-up